helm/chart/teresa/values.yaml

```diff
@@ -30,3 +30,6 @@ build:
     cpu: 500m
     memory: 1024Mi
 debug: false
+useMinio: false
+minio:
+  serviceType: ClusterIP
```

Comment:
why not ELB? are you thinking about minikube?

Reply:
I guess that the `minio` will only be used by `teresa` then I don't need an ELB. Makes sense?

Reply:
I'm not sure about that. (Don't forget you need to create the bucket before use with Teresa)

Reply:
Now this create the bucket